Services/Mail/classes/class.ilMailOptions.php

```diff
@@ -162,7 +157,7 @@ public function updateOptions() : int
     
     public function getLinebreak() : int
     {
-        return $this->linebreak;
+        return self::DEFAULT_LINE_BREAK;
```

Comment:
Why don't we drop `\ilMailOptions::getLinebreak` as well?

Classes to be changed (there are maybe more):
- \ilUserXMLWriter
- \ilUserImportParser
- \ilFormatMail
- \ilMailOptionsTest